graph-db/README.md

```diff
@@ -25,6 +25,12 @@ npm is a separate project from Node.js, and tends to update more frequently. As
 npm install -g npm@latest
 ```
 
+### Serverless
```

Comment:
This is no longer necessary with newer versions of Serverless, adding Serverless to your local package.json should now do the trick.